### DISCUSSION TITLE
Data Collection

### DISCUSSION THREAD

**Discussion by alberto-jlg** [PythonSemicolon/550FinalProject#15] [Ideas]
### Translator Model Data

List of translators:
- Borges, Jorge Luis (Spanish)
- Kruger, Lore (German)
- Haryono, A (Bahasa Indonesia)
- Rajt-Kovaleva, Rita (Russian)
- Murakami, Haruki (Japanese)

For every translator t_i, find:

Let w1, w2, w3, w4, w5 be five different works, each respectively written by a1, a2, a3, a4, a5,
all of which have been translated by translator t_i into language L_i.

TRAINING SET for t_i:
- Translations of w1, w2, w3 by translator t_i.
- Other translations of w1, w2, w3 by any other translator.
- Any other works that have been translated into L_i, by any author and by any translator.

DEV SET for t_i
- Translations of w4 by translated by t_i.
- Other translations of w4 by any other translator.
- Any other works that have been translated into L_i, by any author and by any translator.

TEST SET for t_i: 
- Translations of w5 by translated by t_i.
- Other translations of w5 by any other translator.
- Any other works that have been translated into L_i, by any author and by any translator.

### TODO: Author Model Data
